### PR TITLE
feat: add VSCode debug example

### PR DESCRIPTION
**Pull request by dipasqualew**
A `vscode-template/tasks.json` file will be added, as a debug configuration example to be copied over other repositories.

See:
- [ENG-42]

[ENG-42]: https://comicrelief.atlassian.net/browse/ENG-42

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,11 +14,25 @@ For local development you can simulate lambda and an API endpoint locally using
...
 
 This repository uses JavaScript import aliases with Webpack. If you are using VS Code, the `jsconfig.json` file should handle the autocompletion for you. If you are using WebStorm, you can refer to [this stackoverflow thread](https://stackoverflow.com/questions/34943631/path-aliases-for-imports-in-webstorm).
 
+## Debugging (VS Code)
+
+This repo includes a pre-configured template for the VS Code NodeJS Debugger. It can be found at `.templates/vscode/launch.json`.
+
+To get started with debugging, copy the file to `.vscode/launch.json`, or copy its contents to your existing `.vscode/launch.json` file.
+
+Now you can press `CTRL+SHIFT+D` to open the `Run` tab and execute the `Debug: Serverless Offline` configuration.
```
Comment (corinja) [inline README.md:28]: Seems to be SHIFT+CMD+D, on Mac. Is CTRL on Windows? https://code.visualstudio.com/docs/editor/debugging

Comment (dipasqualew) [reply README.md:28]: Yeah. Windows and Linux. Thanks Corin!

Comment (dipasqualew) [reply README.md]: 79cc0a526274506be6a07883a9f2d3638d62921b

**Review by seb-cr — APPROVED** [review]
Had a play with this yesterday and it works well.

**Pull request merged**